### PR TITLE
Feature/tab bar

### PR DESCRIPTION
**Pull request by sesur**

### CODE REVIEW

File: ToDoList/Modules/AddTaskScreen/Views/AddTaskViewController.swift [hunk 1]
```diff
@@ -23,12 +23,12 @@ class AddTaskViewController: UIViewController, AddTaskScreenDelegate {
...
         
         taskTextField.layer.borderColor = #colorLiteral(red: 0.1607843137, green: 0.1803921569, blue: 0.2352941176, alpha: 1)
-        taskTextFieldView.layer.cornerRadius = 10
+        taskTextFieldView.layer.cornerRadius = Constants.cornerRadiusValue
         
         createButtons()
         arangeButtons()
         
-        createButton.setGradientBackground(startColor: UIColor(named: "Dark Purple")!, endColor: UIColor(named: "Light Purple")!)
+        createButton.setGradientBackground(startColor: UIColor(named: ColorNames.darkPurple)!, endColor: UIColor(named: ColorNames.lightPurple)!)
```
Comment (vvornic) [inline ToDoList/Modules/AddTaskScreen/Views/AddTaskViewController.swift:31]: Also you could try this instead:
https://medium.com/ios-os-x-development/ios-extend-uicolor-with-custom-colors-93366ae148e6

File: ToDoList/SceneDelegate.swift [hunk 2]
```diff
@@ -16,7 +16,10 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         guard let windowScene = (scene as? UIWindowScene) else { return }
         window = UIWindow(frame: windowScene.coordinateSpace.bounds)
         window?.windowScene = windowScene
-        window?.rootViewController = ContainerViewController(child: TableViewController())
+        window?.rootViewController = TabBarViewController()
+        UINavigationBar.appearance().titleTextAttributes = [NSAttributedString.Key.foregroundColor: UIColor.white]
+        
+        // ContainerViewController(child: TableViewController())
```
Comment (vvornic) [inline ToDoList/SceneDelegate.swift:22]: remove this comment

Comment (sesur) [reply ToDoList/SceneDelegate.swift:22]: done

File: ToDoList/Modules/AddTaskScreen/Views/AddTaskViewController.swift (hunk 1 above)
Comment (sesur) [reply ToDoList/Modules/AddTaskScreen/Views/AddTaskViewController.swift]: ok, I will

File: ToDoList/Modules/Settings/View/SettingsViewController.swift [hunk 3]
```diff
@@ -23,6 +23,11 @@ class SettingsViewController: UIViewController, SettingsView {
         presenter = SettingsViewPresenterImp(view: self)
     }
     
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(true)
+        title = ViewControllerNames.settingsTitle
```
Comment (vtoloconnicov) [inline ToDoList/Modules/Settings/View/SettingsViewController.swift:28]: who's title is setting here? this method is redundant

File: ToDoList/Resources/Constants.swift [hunk 4]
```diff
@@ -35,3 +35,28 @@ struct ColorNames {
     static let workColor = "Work"
     static let generalColor = "General"
 }
+
+struct ViewControllerNames {
+
```
Comment (vtoloconnicov) [inline ToDoList/Resources/Constants.swift:40]: space

File: ToDoList/Resources/Constants.swift [hunk 5]
```diff
@@ -35,3 +35,28 @@ struct ColorNames {
...
+    static let gearShape = "gearshape"
+    static let bicycle = "bicycle"
+    static let heart = "heart"
+    static let briefCase = "briefcase"
+    static let scribbleVariable = "scribble.variable"
+    static let bag = "bag"
+}
+
+struct CategoryName {
+
```
Comment (vtoloconnicov) [inline ToDoList/Resources/Constants.swift:56]: space

File: ToDoList/SceneDelegate.swift [hunk 6]
```diff
@@ -16,7 +16,10 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         guard let windowScene = (scene as? UIWindowScene) else { return }
         window = UIWindow(frame: windowScene.coordinateSpace.bounds)
         window?.windowScene = windowScene
-        window?.rootViewController = ContainerViewController(child: TableViewController())
+        window?.rootViewController = TabBarViewController()
+        UINavigationBar.appearance().titleTextAttributes = [NSAttributedString.Key.foregroundColor: UIColor.white]
```
Comment (vtoloconnicov) [inline ToDoList/SceneDelegate.swift:20]: is this necessary?

File: ToDoList/Modules/AddTaskScreen/Views/AddTaskViewController.swift (hunk 1 above)
Comment (vtoloconnicov) [reply ToDoList/Modules/AddTaskScreen/Views/AddTaskViewController.swift]: we will try it in next project

File: ToDoList/SceneDelegate.swift (hunk 6 above)
Comment (sesur) [reply ToDoList/SceneDelegate.swift:20]: already not, removed

File: ToDoList/Resources/Constants.swift (hunk 5 above)
Comment (sesur) [reply ToDoList/Resources/Constants.swift:56]: fixed

File: ToDoList/Resources/Constants.swift (hunk 4 above)
Comment (sesur) [reply ToDoList/Resources/Constants.swift:40]: removed

File: ToDoList/Modules/Settings/View/SettingsViewController.swift (hunk 3 above)
Comment (sesur) [reply ToDoList/Modules/Settings/View/SettingsViewController.swift:28]: yes, removed

File: ToDoList/Modules/AddTaskScreen/Views/AddTaskViewController.swift (hunk 1 above)
Comment (sesur) [reply ToDoList/Modules/AddTaskScreen/Views/AddTaskViewController.swift]: deal, good stuff

File: ToDoList/UIUtils/ContainerViewController.swift [hunk 7]
```diff
@@ -32,23 +32,20 @@ class ContainerViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         updateContainerView()
+        title = ViewControllerNames.homeTitle
```
Comment (vtoloconnicov) [inline ToDoList/UIUtils/ContainerViewController.swift:35]: what title is that?

Comment (sesur) [reply ToDoList/UIUtils/ContainerViewController.swift:35]: dummy title, removed

**Review by vtoloconnicov — APPROVED** [review]

**Pull request merged**